### PR TITLE
Update Occitan language locale file

### PR DESCRIPTION
**Pull request by Mejans**

### CODE REVIEW

**Comment by Mejans** [comment]
I did sign

**Review by jpnurmi — COMMENTED** [review]
> I did sign

The e-mail on GitHub doesn't match the email address on Launchpad?

File: packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb [hunk 1]
```diff
@@ -47,5 +47,22 @@
   "diskHeadersSize": "Talha",
   "diskHeadersUsed": "Utilizat",
   "diskHeadersSystem": "Sistèma",
-  "diskHeadersFormat": "Formatar"  
-}
+  "diskHeadersFormat": "Formatar"
```
[the remark this answers is not in the document]
Comment (Mejans) [reply packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb:50]: I've just added the address. Thanks

**Comment by MarcusTomlinson** [comment]
> I did sign

It can take a couple days to be added to the GitHub group.

File: packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb [hunk 2]
```diff
@@ -39,13 +39,30 @@
...
+  "writeChangesFallbackSerial": "disc",
+  "writeChangesDescription": "Se contunhatz, las modificacions çai-jos seràn escritas suls disques. Poiretz realizar de cambiaments mai tard a la man.",
+  "writeChangesPartitionTablesHeader": "Las taulas de particions dels periferics seguents seràn modificadas :",
+  "writeChangesPartitionTablesEntry": "{serial} ({path})",
+  "writeChangesPartitionsHeader": "Las particions seguentas seràn formatadas :",
+  "writeChangesPartitionEntryPrimaryFull": "particion n° {partitionNumber} de {diskSerial} ({diskPath}) de tipe {fstype} coma {mountPath}",
+  "writeChangesPartitionEntryPrimary": "particion n° {partitionNumber} de {diskSerial} ({diskPath}) coma",
+  "writeChangesPartitionEntrySecondary": "        particion n° de tipe {fstype} coma {mountPath}",
+
+  "chooseYourLookPageTitle": Causissètz l'aparéncia",
```
Comment (oSoMoN) [inline packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb:62]: That string is missing an opening double quote.

**Review by oSoMoN — CHANGES_REQUESTED** [review]

File: packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb (hunk 2 above)
Comment (Mejans) [reply packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb]: changed!

File: packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb (hunk 1 above)
[the remark this answers is not in the document]
Comment (oSoMoN) [reply packages/ubuntu_desktop_installer/lib/l10n/app_oc.arb]: Thanks!

**Comment by oSoMoN** [comment]
Thanks!

**Pull request merged**